template_scripts/distribution.sh

```diff
@@ -10,6 +10,11 @@ if [ -n "${REPO_PROXY}" ]; then
     YUM_OPTS="$YUM_OPTS --setopt=proxy=${REPO_PROXY}"
 fi
 
+if [ -n "${FEDORA_MIRRORS}" ]; then
+    YUM_OPTS="$YUM_OPTS --setopt=fedora.baseurl=${FEDORA_MIRRORS}/fedora/linux/releases/${DIST/fc/}/Everything/x86_64/os/"
```

Comment:
Oh, but this actually expect just one mirror, not a list of them. So, either get just first entry (word) of FEDORA_MIRRORS (and care about supporting multiple of them later, if at all). Or rename to FEDORA_MIRROR...
Maybe just add something like this at the top:
```
FEDORA_MIRROR=$(echo "$FEDORA_MIRRORS" | awk '{print $1}')
```

Reply:
Supporting multiple mirrors would it be necessary/useful? Indeed, we can let this possibility and do as you suggest.

Reply:
I wouldn't implement multiple mirrors right now. But it could be useful to have some fallback automatically. For example the most common reason by build fails on travis if failing to synchronize repository metadata...

Reply:
Ok. So I rename everything to FEDORA_MIRROR. The best next step would be to generate a metalink xml file from a list of URLS. Do you agree?

Reply:
👍